PV260_Project/BusinessLayer/Jobs/DataSyncJob.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BusinessLayer.DataLoading;
+using DataLayer.Models;
+using BusinessLayer.Services;
+using BusinessLayer.Exceptions;
+
+namespace BusinessLayer.Jobs
+{
+    public class DataSyncJob : IDataSyncJob
+    {
+        private readonly IDataDownloader _downloader;
+        private readonly IDataSetService _dataSetService;
+        private readonly IFileUrlService _fileUrlService;
+
+        private Timer? timer;
+
+        public DataSyncJob(IDataDownloader downloader, IDataSetService dataSetService, IFileUrlService fileUrlService)
+        {
+            _downloader = downloader;
+            _dataSetService = dataSetService;
+            _fileUrlService = fileUrlService;
+        }
+
+        public void Run()
+        {
+            timer = new Timer(new TimerCallback(DownloadAndSaveFile), null, 0, 24 * 60 * 60 * 1000);

```

Comment:
Since we will be evaluated by the quality of our code, we should not use just bare numbers. The better solution would be to create constant with an appropriate name and pass it to this method.

Reply:
added constant for the interval